resources/mediaqueries.css

```diff
@@ -0,0 +1,29 @@
+/* The ones I use most fequently */
+
+/* min 768px // iPad & Up */
+@media screen and (min-width: 768px) {
+}
+
+/* Min 1200px // Desktop & Up */
+@media screen and (min-width: 1200px) {
+}
+
+/****** Other helpful breakpoints ******/
```

Comment:
@murder-cat These are the ones I use for any edge cases / fancy-pants UI requirements